omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -129,13 +129,11 @@
   postgresql
   oc_bifrost
   oc_id
```

Comment:
Might be worth cleaning up the associated recipes the removed items.
* opscode-expander.rb
* rabbitmq.rb

Why does opscode-solr4 remain?

Reply:
opscode-solr4 remains to support the external-solr=elasticsearch setup. That should also be changed eventually to something along the lines of external-elasticsearch.

I could potentially use this PR to remove the attributes and recipes. But if merging this in would mean that the build in current would not have the old java, that might help with a lot of pain. And I can remove the attributes, recipes and eventually the unused code as part of separate PR's, if that seems ok.

Reply:
#2061 and #2062